### PR TITLE
adding solution to problem 315 - Count of Smaller Number After Self in Python

### LINKED ISSUE #810: 315 | Count of Smaller Numbers After Self | Python
Problem name: Count of Smaller Numbers After Self
Difficulty: Hard
Language: Python

### PR DESCRIPTION
**Pull request by SousaPedro11**
Fix #810

### CODE REVIEW

**Comment by SousaPedro11** [comment]
The algorithm works as follows:
If the entry is nums = [5, 2, 9, 7, 1, 6]
the sorted is [1, 2, 5, 6, 7, 9]
the initial result is [0, 0, 0, 0, 0, 0]

In the loop, I analyze the input list from its corresponding indexes and values

for example:
In the first iteration, the index is 0 and the value is 5.
So I search for the first index of that value in the ordered list, assign it to the corresponding index in the results list, and then remove the element from the ordered list.
The result becomes [2, 0, 0, 0, 0, 0], and sorted lists becomes [1, 2, 6, 7, 9] (without 5) after the first iteration.

So basically the index of the ordered list shows the quantity of the smallest values after the analyzed element of the input list.
